### PR TITLE
Welcome page

### PR DESCRIPTION
**Pull request by s-kond**
#User Story 1: Welcome page

### CODE REVIEW

**Review by pmarx92 — APPROVED** [review]
looks good

**Review by dorosiemers — APPROVED** [review]

**Review by nazardiachok — APPROVED** [review]

**Review by VladF146 — APPROVED** [review]

**Review by BektasTu — APPROVED** [review]

**Review by kornrumpfl — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,73 +1,30 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Your React {state.topic} Template
-        </AppLink>
-      </AppHeader>
+      <StyledHeader>
+        <h1>Cook4All</h1>
+      </StyledHeader>
```
Comment (mpagels) [inline src/App.js:8]: Do you need the `StyledHeader` here? Why not style the `h1` itself?

File: src/App.js [hunk 2]
```diff
@@ -1,73 +1,30 @@
...
+      <p>Welcome!</p>
```
Comment (mpagels) [inline src/App.js]: This could be semantically a `h2` element.

File: src/App.js [hunk 3]
```diff
@@ -1,73 +1,30 @@
...
-  @keyframes App-logo-spin {
-    from {
-      transform: rotate(0deg);
-    }
-    to {
-      transform: rotate(360deg);
-    }
-  }
+const StyledHeader = styled.header`
+  background-color: #dcedc1;
```
Comment (mpagels) [inline src/App.js]: You could put the hex code in an root variable as well.

**Review by mpagels — APPROVED** [review]
Very good start here Stefan, 

found some small things. Pls review,

File: src/App.js (hunk 1 above)
Comment (s-kond) [reply src/App.js:8]: Because I want to add a Menu or at least an Icon to the Header later.

File: src/App.js (hunk 2 above)
Comment (s-kond) [reply src/App.js]: Right 👍

File: src/App.js (hunk 3 above)
Comment (s-kond) [reply src/App.js]: 👍

**Pull request merged**